react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.stories.tsx

```diff
@@ -320,3 +323,99 @@ export const ContinuousColorTable: React.FC = () => {
         </>
     );
 };
+
+// colorselector for welllayer
+const wellLayers = [
+    {
+        ...defaultProps.layers[0],
+        refine: false,
+        outline: false,
+        logData: "./volve_logs.json",
+        logrunName: "BLOCKING",
+        logName: "ZONELOG",
+        logColor: "Stratigraphy",
+        colorMappingFunction: createColorMapFunction("Stratigraphy"),
+    },
+];
+
+// prop for legend
+const min = 0;
+const max = 0.35;
+const dataObjectName = "ZONELOG";
+const position = [16, 10];
+const horizontal = true;
+const colorName = wellLayers[0].logColor;
+const discreteData = {
+    Above_BCU: [[], 0],
+    ABOVE: [[], 1],
+    H12: [[], 2],
+    H11: [[], 3],
+    H10: [[], 4],
+    H9: [[], 5],
+    H8: [[], 6],
+    H7: [[], 7],
+    H6: [[], 8],
+    H5: [[], 9],
+    H4: [[], 10],
+    H3: [[], 11],
+    H2: [[], 12],
+    H1: [[], 13],
+    BELOW: [[], 14],
+};
+
+//eslint-disable-next-line
+const wellLayerTemplate = (args: any) => {
+    const [wellLegendUpdated, setWellLegendUpdated] = React.useState();
+
+    const wellLayerData = React.useCallback((data) => {
+        setWellLegendUpdated(data);
+    }, []);
+
+    const layers = [
+        {
+            ...args.wellLayers[0],
+            colorMappingFunction: createColorMapFunction(
+                wellLegendUpdated ? wellLegendUpdated : wellLayers[0].logColor
+            ),
+            logColor: wellLegendUpdated
+                ? wellLegendUpdated
+                : wellLayers[0].logColor,
+        },
+    ];
+
+    return (
+        <div>
+            <div
+                style={{
+                    float: "right",
+                    zIndex: 999,
+                    opacity: 1,
+                    position: "relative",
+                }}
+            >
+                <ColorLegend {...args} getColorMapname={wellLayerData} />
+            </div>
+            <DeckGLMap {...args} layers={layers} />
+        </div>
+    );
+};
+
+//eslint-disable-next-line
+export const WellLayerColorSelector: any = wellLayerTemplate.bind({});
+
+WellLayerColorSelector.args = {
```

Comment:
The example is too small in the Docs page: 

![Screenshot from 2022-06-14 10-12-36](https://user-images.githubusercontent.com/19621822/173528601-4bde2b61-6a0d-4155-80f9-b71dfb16b5c9.png)

I think it would also be useful with a description that lets you know how to change the color scale.